package/CHANGELOG

```diff
@@ -18,6 +18,7 @@ The rules for this file:
  * 2.6.0
 
 Fixes
+  * Fix Boltzmann typo in `units.py` (PR #4214, Issue #4213)
```

Comment:
Actually, you need to add your handle in the author list for `2.6.0`.

Reply:
Whoops, silly oversight. Thanks, added!